sdk/Sdk.Generators/Sdk.Generators.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <LangVersion>latest</LangVersion>
+    <PackageId>Microsoft.Azure.Functions.Worker.Sdk.Generators</PackageId>
+    <Description>This package provides source generation for azure function worker.</Description>
```

Comment:
Nit: Can we make this description more like [this one](https://github.com/Azure/azure-functions-dotnet-worker/blob/f6c139f1990b3b94fb61408aa09090bd668c927b/sdk/Sdk.Analyzers/Sdk.Analyzers.csproj#L10)? It could be something like "This package provides source generation for the Azure Functions .NET Worker."